### PR TITLE
RC Security reorg: Split CIDR allow list into a Database CIDIR allow list and Subscription CIDR allow list

### PR DESCRIPTION
**Pull request by cmilesb**
Ticket: DOC-2703
Staging link: 
https://redis.io/docs/staging/DOC-2703/operate/rc/security/cidr-whitelist/
https://redis.io/docs/staging/DOC-2703/operate/rc/cloud-integrations/aws-cloud-accounts/subscription-whitelist

### CODE REVIEW

File: content/operate/rc/security/cidr-whitelist.md [hunk 1]
```diff
@@ -13,15 +13,13 @@ weight: 30
 
 The [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) [allow list](https://en.wikipedia.org/wiki/Whitelist) lets you restrict traffic to your Redis Cloud database. When you configure an allow list, only the [IP addresses](https://en.wikipedia.org/wiki/IP_address) defined in the list can connect to the database. Traffic from all other IP addresses is blocked.
 
-## Database allow list
-
 You can configure your database's CIDR allow list to restrict client connections to a specific range of IP addresses.
```
Comment (andy-stark-redis) [inline content/operate/rc/security/cidr-whitelist.md:16]: This reads like it just repeats what was said in the paragraph above. If the idea is to emphasise that you can specify a range (ie, with a start and end point) as well as individual IP addresses then you could probably just add that to the previous paragraph.

**Review by andy-stark-redis — COMMENTED** [review]
One possible issue but otherwise, the language and links look good.

File: content/operate/rc/security/cidr-whitelist.md (hunk 1 above)
Comment (cmilesb) [reply content/operate/rc/security/cidr-whitelist.md:16]:
```suggestion
```

Makes sense. Thanks!

**Review by andy-stark-redis — APPROVED** [review]
Language looks good (but I don't know enough about the actual info to verify it :-) ).

**Pull request merged**